desktop/apps/auction/routes.coffee

```diff
@@ -48,7 +48,6 @@ setupUser = (user, auction) ->
 
   artworks.comparator = (artwork) ->
     saleArtwork = artwork.related().saleArtwork
-    saleArtwork.get('lot_number') or
```

Comment:
After importing lots, the ops team sets `position` manually in ohm to determine lot order for questionable lots, leaving the `label` field open to whatever they want.